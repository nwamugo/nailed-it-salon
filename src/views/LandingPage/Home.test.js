import React from 'react';
import { shallow } from 'enzyme';
import Home from './index';

describe('Home', () => {
  it('should render correctly', () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
