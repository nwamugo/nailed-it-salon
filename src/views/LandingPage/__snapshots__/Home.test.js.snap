// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should not regress 1`] = `
<div>
  <Welcome />
  <div
    className="content"
  >
    <Services />
    <OurStory />
    <ContactSection />
  </div>
  <Footer />
</div>
`;
